packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/RelationModal.tsx

```diff
@@ -333,94 +334,96 @@ const RelationModal = ({ children }: { children: React.ReactNode }) => {
   const isCreating = useRelationModal('RelationModalForm', (state) => state.isCreating);
 
   return (
-    <Modal.Root
-      open={state.isModalOpen}
-      onOpenChange={(open) => {
-        if (!open) {
-          dispatch({
-            type: 'CLOSE_MODAL',
-            payload: { shouldBypassConfirmation: false },
-          });
-        }
-      }}
-    >
-      {children}
-      <StyledModalContent>
-        <Modal.Header gap={2}>
-          <Flex justifyContent="space-between" alignItems="center" width="100%">
-            <Flex gap={2}>
+    <ComponentProvider id={undefined} level={-1} uid={undefined} type={undefined}>
```

Comment:
We need a comment to explain why this is necessary

Reply:
ok sure